data/wanderer/wanderers start.txt

```diff
@@ -70,6 +79,10 @@ mission "First Contact: Wanderer"
 				`	"Sorry, I don't speak their language."`
 				`	"Are there any Hai here?"`
 			`	The same alien speaks again, pantomiming your ship flying into the sky and then returning, and then points to your airlock and mimics your rodent-ear gesture for the Hai. You think they are telling you to go find one of the Hai who can serve as an interpreter, and bring them back here.`
+			branch accept
+				not "<uhai> 18: Hai Language: active"
+			`	You should probably go back to the Unfettered and speak to them about the need for a diplomat, or any way to understand these aliens. He-Yi should be able to help you with this.`
+			`	Alternatively, you could go to the Hai and ask them for a translator instead, although He-Yi will probably respect you less for it.`
```

Comment:
This would break the accept below exczpt if I add an empty line
I wonder, does it not work? Caus branch to accept would be useful, warp told me it'd work iirc

Reply:
It might work, but I'm not sure it does.
Hence the suggestion.

Reply:
Well suggestion wont work, at least I think you cant have an accept hanging on a label